proposals/lip-0049.md

```diff
@@ -206,12 +204,15 @@ Consider a binary message `ccmBytes`, corresponding to a serialized cross-chain
 
 ```python
 def decodeCCM(ccmBytes: bytes) -> CCM:
```

Comment:
This function is not used anywhere. We could remove it.

Reply:
I would also keep this function for consistency.